src/ResourceManager/Network/Commands.Network/LoadBalancer/InboundNatPool/GetAzureLoadBalancerInboundNatPoolConfigCommand.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRMLoadBalancerInboundNatPoolConfig"), OutputType(typeof(PSInboundNatPool))]
```

Comment:
Fixed in the latest commit. (https://github.com/ankurchoubeymsft/azure-powershell/commit/e6cb77ad5e8e7d3875507ab8f08772cd09d8ea83)